src/python/WMCore/WMSpec/Steps/Executor.py

```diff
@@ -74,6 +74,7 @@ def __init__(self):
         self.workload = None
         self.job = None
         self.errorDestination = None
+        self.failedPreviousStep = None
```

Comment:
Isn't this only needed for the CMSSW module? If so, then I'd suggest to remove it from here.

Reply:
Yes, but pylint complained this wasn't declared in `__init__` as a "must fix" item.

Reply:
Don't trust everything pylint says ;) AFAIR, one classical case where it gets lost is when we set object attributes with `setattr`

Reply:
regarding waiting to have a completed job, I haven't yet, still waiting :). I just injected a couple of extra different workflows that I know usually finish with no errors to make sure things are good.
Regarding the pylint thing... any idea on how to get rid of  `self.failedPreviousStep` in `__init__` and not have the pylint report as "failed"? Unless the failed report doesn't matter for this particular case.

Reply:
We could disable it, but I usually prefer to have these complaints than disable things.
BTW, is it complaining in the jenkins test or locally? I just ran pylint (1.9.4) on my laptop and didn't see such complain.

Reply:
@amaltaro I can't make DAS queries via cmsweb-testbed.cern.ch, but PhEDEX is showing 
 
[test1](https://cmsweb-testbed.cern.ch/phedex/prod/Data::Replicas?bexp=11921801;filter=%20%2FRelValSinglePiE50HCAL%2FDMWM-TC_SLC7_khurtado_slc7test_cv1-v11%2FGEN-SIM;dexp=1420163#b11921801
)

[GEN-SIM in test2](/RelValProdMinBias/DMWM-StepChain_DupOutMod_khurtado_stepchaindup_v9-v11/GEN-SIM)

Reply:
Good! I guess I haven't mentioned, but we have this script doing some reasonable checks for us:
https://github.com/dmwm/WMCore/wiki/Injecting,-assigning-and-validating-test-requests#validating-requests